docs/docs/core/fallback-actions.mdx

```diff
@@ -6,117 +6,146 @@ description: Define custom fallback actions with thresholds for NLU and Core for
 ---
 
 Sometimes you want to revert to a fallback action, such as replying,
-“Sorry, I didn’t understand that”. You can handle fallback cases by adding
-either the `FallbackPolicy` or the `TwoStageFallbackPolicy` to your
-policy ensemble.
-
-## Fallback Policy
-
-The `FallbackPolicy` has one fallback action, which will
-be executed if the intent recognition has a confidence below `nlu_threshold`
-or if none of the dialogue policies predict an action with
-confidence higher than `core_threshold` or if the highest ranked intent differs in
-confidence with the second highest ranked intent by less than `ambiguity_threshold`.
-
-The thresholds and fallback action can be adjusted in the policy configuration
-file as parameters of the `FallbackPolicy`.
+`"Sorry, I didn't understand that"`. You can handle fallback cases by adding appropriate
+rules. Rasa Open Source comes with two default implementations for handling these
+fallbacks.
+In addition, you can also use [Custom Actions](./actions.mdx#custom-actions) to run any
+custom code.
+
+## Handling Low NLU Confidence
+
+Although Rasa's [Intent Classifier](../nlu/components.mdx#intent-classifiers) will
```

Comment:
I found out that relative links sometimes break, does it pass the build?

Reply:
yep, passes 👍 Should I change it to absolute links anyhow?

Reply:
no if they do not break, keep it like that. I need to figure out why it didn't work before, not even sure I can repro now